boot/boot_test.go

```diff
@@ -563,13 +563,31 @@ func (s *bootSetSuite) TestMakeBootable20MultipleRecoverySystemsError(c *C) {
 
 func (s *bootSetSuite) TestMakeBootable20RunMode(c *C) {
 	dirs.SetRootDir("")
+	bootloader.Force(nil)
 
 	model := makeMockUC20Model()
 	rootdir := c.MkDir()
 	seedSnapsDirs := filepath.Join(rootdir, "/snaps")
 	err := os.MkdirAll(seedSnapsDirs, 0755)
 	c.Assert(err, IsNil)
 
+	// grub on ubuntu-seed
+	runMnt := filepath.Join(rootdir, "/run/mnt/")
```

Comment:
unrelated, we defined now in one of the PRs  dirs.RunMnt ?  we should start using it where it makes sense, not so important for tests, but in the prod code?